bytecomp/translcore.ml

```diff
@@ -438,31 +438,33 @@ let transl_primitive loc p env ty path =
   match prim with
   | Plazyforce ->
       let parm = Ident.create "prim" in
-      Lfunction{kind = Curried; params = [parm];
+      Lfunction{kind = Curried; params = [parm, Pgenval]; return = Pgenval;
                 body = Matching.inline_lazy_force (Lvar parm) Location.none;
                 loc = loc;
-                attr = default_stub_attribute }
+                attr = default_function_attribute }
```

Comment:
Oups. good catch.

This patch was extracted from a sequence started before the merge of the stub patch.
I will fix this